client/components/organisms/SettingsRegister/SettingsRegister.js

```diff
@@ -93,7 +92,7 @@ export default function Register() {
  
  
   const resetState = () => {
-    setUsername(user.username);
+    setFullname(user.full_name);
     setPhoneNumber(user.phone_number);
```

Comment:
And where is the part that updates the email to be the username ?

Reply:
plus, the email input should be disabled

Reply:
I've changed it to username and the email input already disabled